src/ert/parsing/lark_parser.py

```diff
@@ -250,18 +252,57 @@ def _substitute_args(
     ]
 
 
+class IncludeNode:
```

Comment:
We have a bit of PSD when it comes to naming things with "Node". Its too general and undescriptive. I suggest we call it `IncludedFile` and rename `parent` to `included_from`.